modules/tinymce/src/themes/mobile/main/ts/api/Settings.ts

```diff
@@ -6,14 +6,26 @@
  */
 
 import Editor from 'tinymce/core/api/Editor';
+import DefaultStyleFormats from '../features/DefaultStyleFormats';
 
-const isSkinDisabled = (editor: Editor): boolean => editor.settings.skin === false;
+const defaults = [ 'undo', 'bold', 'italic', 'link', 'image', 'bullist', 'styleselect' ];
+
+const isSkinDisabled = (editor: Editor): boolean => editor.getParam('skin') === false;
 
 const readOnlyOnInit = (_editor: Editor) =>
-  // Intentional short circuit, TODO: implement editor.settings.mobile
+  // Intentional short circuit, TODO: implement editor.getParam('mobile')
```

Comment:
This is worryingly outdated. `mobile` is a setting now but it's not for the mobile **theme** (which is an old concept). This is also just a stub of a method... I think this should be changed to something like `Intentional short circuit because stub. Was meant to reference a setting for readonly on init in mobile theme.`? Should probably get opinions from others too. I just really dislike how potentially misleading this is, so I'd like to change it even though it's outside the scope of your ticket.

Reply:
I think maybe the original intention here isn't what you're thinking. I'm pretty sure this just meant to check `editor.settings.mobile.readonly` so as you could make the editor readonly on mobile specifically.

Reply:
In saying that though, we're never going to fix this now, so maybe just delete the content after todo?